reviews_test.rb

```diff
@@ -0,0 +1,121 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './employee_reviews'
+require './employee'
+require './department'
+require "pry-byebug"
+
+
+class ReviewsTest < Minitest::Test
+
+  def test_employee_class_exists
+    assert Employee
+  end
+
+  def test_department_class_exists
+    assert Department
+  end
+
+  def test_create_new_employee
+    employee = Employee.new("Anna", 50000)
+    assert employee.name == "Anna"
+  end
+
+  def test_create_new_employee_salary
+    employee = Employee.new("Anna", 50000)
+    assert employee.salary == 50000
+  end
+
+  def test_department_name
+    department = Department.new("Accounting")
+    assert department.dept_name == "Accounting"
+  end
+
+  def test_employee_go_to_department
+    anna = Employee.new("Anna", 50000)
+    accounting = Department.new("Accounting")
+    accounting.add_employee(anna)
+    assert accounting.employees.include?(anna)
+  end
+
+  def test_get_employee_name
+    anna = Employee.new("Anna", 50000)
+    bob = Employee.new("Bob", 60000)
+    clarissa = Employee.new("Clarissa", 75000)
```

Comment:
Cool.  You may have overdone it a bit by having Clarissa here when she's not used, but it's a minor thing.